src/utils/connect.js

```diff
@@ -0,0 +1,115 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+import * as React from 'react';
+import { connect } from 'react-redux';
+import type {} from 'react-redux';
```

Comment:
what's this line for ?

Reply:
For fun ;)
I guess it's leftover. I removed it.